### ISSUE TITLE
Deprecated ability to execute multiple alias mutations atomically

### ISSUE THREAD

**Issue by luismfonseca** [sksamuel/elastic4s#421]
I've experiencing the same difficulty as described on issue #140. But I see that the latest version has the follow code marked as deprecated:

```
esClient.execute(
   aliases(remove alias aliasIndexName on index, add alias aliasIndexName on id))
```

The `bulk` operation doesn't support aliases, so my question is: what's the new way of doing things?

**Comment by sksamuel** [comment]
Which bit is deprecated?

**Comment by luismfonseca** [comment]
The `case object aliases` bit: https://github.com/sksamuel/elastic4s/blob/b96f3aee5f91baecd745bc993a59f090a26895ce/src/main/scala/com/sksamuel/elastic4s/ElasticDsl.scala#L40.

I've noticed that the latest version doesn't have the `case object`. Also, I've noticed that the IntelliJ is confusing the `case object aliases` with the `def aliases` and marking my code as using a deprecated function incorrectly. So I think it's safe to close the issue :o.
